test/functional/options/defaults_spec.lua

```diff
@@ -405,13 +409,15 @@ describe('XDG defaults', function()
 
   describe('with XDG variables that can be expanded', function()
     before_each(function()
-      clear({env={
-        XDG_CONFIG_HOME='$XDG_DATA_HOME',
-        XDG_CONFIG_DIRS='$XDG_DATA_DIRS',
-        XDG_DATA_HOME='$XDG_CONFIG_HOME',
-        XDG_RUNTIME_DIR='$XDG_RUNTIME_DIR',
-        XDG_STATE_HOME='$XDG_CONFIG_HOME',
-        XDG_DATA_DIRS='$XDG_CONFIG_DIRS',
+      clear({
+        args_rm={'runtimepath'},
```

Comment:
Here and in `startup_spec.lua`, remove the `set runtimepath^=./build/lib/nvim/` added in this PR (which is necessary to find parsers not installed on the system).